File: Dataset/Leetcode/train/66/328.py
 class Solution:
    def XXX(self, digits: List[int]) -> List[int]:
        s = digits[::-1]
        i = 0
        if s[0]+1 < 10:       
            s[0] += 1
        else:
            while s[i]+1 >= 10:
                s[i] = 0
                i += 1
                if i != len(s):
                    s[i] += 1
                    if s[i] == 9:
                        break
                else:
                    s.append(1)
           
        s1 = s[::-1]
        return s1

File: Dataset/Leetcode/train/11/328.java
class Solution {
    public int XXX(int[] height) {
         if(height.length==1){
            return 0;
        }
        int max=0;
        for(int i=0,j=height.length-1;i<height.length&&j>i;){
            int area=(j-i)*Math.min(height[i],height[j]);
            max=Math.max(max,area);
            if(height[i]>height[j]){
                j--;
            }else{
                i++;
            }
        }
        return max;
    }
}

